### ISSUE TITLE
FinalInternalClassFixer makes Doctrine entities final when using attributes

### ISSUE THREAD

**Issue by enumag** [PHP-CS-Fixer/PHP-CS-Fixer#7033]
### Description

FinalInternalClassFixer correctly ignores Doctrine entities when they're configured using Doctrine annotations.
However it doesn't ignore them if they're configured using PHP 8 attributes.

### Runtime version

3.17.0

### Code snippet that reproduces the problem

Basic example from Doctrine documentation will suffice.

 ```php
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'products')]
class Product
{
    #[ORM\Id]
    #[ORM\Column(type: 'integer')]
    #[ORM\GeneratedValue]
    private int|null $id = null;
    #[ORM\Column(type: 'string')]
    private string $name;

    // .. (other code)
}
```

**Comment by Wirone** [comment]
Thank you for raising an issue, but It is already reported in #5782 and will be fixed in #6893.

**Comment by enumag** [comment]
@Wirone Ah sorry for double reporting then... I was looking for an open issue but didn't find it because the fixer was renamed since the issue was reported.
